external/openembedded-layer/recipes-multimedia/v4l2apps/v4l-utils_%.bbappend

```diff
@@ -20,6 +20,6 @@ FILES_libv4l:remove:tegra = "${libdir}/libv4l/plugins/*.so"
 
 TEGRA_PLUGINS ?= ""
 TEGRA_PLUGINS:tegra = "tegra-libraries-multimedia-v4l"
-RRECOMMENDS:libv4l += "${TEGRA_PLUGINS}"
+RRECOMMENDS:libv4l:tegra += "${TEGRA_PLUGINS}"
```

Comment:
Is this change needed?  For non-tegra platforms, the value of `TEGRA_PLUGINS` will be null anyway.